vip-cache-manager/vip-cache-manager.php

```diff
@@ -215,6 +215,11 @@ public function execute_purges() {
 			return;
 		}
 
+		// Cache purging disabled, bail
+		if ( defined( 'VIP_GO_DISABLE_CACHE_PURGING' ) && true === VIP_GO_DISABLE_CACHE_PURGING ) {
+			return;
```

Comment:
Should we leave an `error_log` in there (like in the PR description) for added visibility?

Reply:
The volume could be quite large for the use case we're adding this for (certain imports)...

Reply:
Makes sense, thanks for taking a look!

Reply:
Added with 05d64815 . Totally open to reword the message, not 100% sure about it.

Reply:
Not logging anything anymore because we have chosen to put the code in `init()` and that would make the log flooded with cache purging messages.